src/com/lilithsthrone/game/dialogue/utils/PhoneDialogue.java

```diff
@@ -3072,75 +3072,66 @@ public Response getResponse(int responseTab, int index) {
 				return new ResponseEffectsOnly("15 minutes", "Loiter in this area for the next fifteen minutes.") {
 					@Override
 					public void effects() {
-//						Main.mainController.openPhone();
-						Main.game.getTextStartStringBuilder().append(
-								"<p style='text-align:center;'>"
-										+ "<i>You spend the next fifteen minutes loitering about, doing nothing in particular...</i>"
-								+ "</p>");
-						Main.game.endTurn(60*15);
-						Main.game.setContent(new Response("", "", Main.game.getDefaultDialogue()));
+						Loiter(15);
 					}
 				};
 				
 			} else if(index == 2) {
 				return new ResponseEffectsOnly("1 hour", "Loiter in this area for the next hour.") {
 					@Override
 					public void effects() {
-//						Main.mainController.openPhone();
-						Main.game.getTextStartStringBuilder().append(
-								"<p style='text-align:center;'>"
-										+ "<i>You spend the next hour loitering about, doing nothing in particular...</i>"
-								+ "</p>");
-						Main.game.endTurn(60*60);
-						Main.game.setContent(new Response("", "", Main.game.getDefaultDialogue()));
+						Loiter(60);
 					}
 				};
 				
 			} else if(index == 3) {
 				return new ResponseEffectsOnly("4 hours", "Loiter in this area for the next four hours.") {
 					@Override
 					public void effects() {
-//						Main.mainController.openPhone();
-						Main.game.getTextStartStringBuilder().append(
-								"<p style='text-align:center;'>"
-										+ "<i>You spend the next four hours loitering about, doing nothing in particular...</i>"
-								+ "</p>");
-						Main.game.endTurn(60*60*4);
-						Main.game.setContent(new Response("", "", Main.game.getDefaultDialogue()));
+						Loiter(60*4);
 					}
 				};
 				
 			} else if(index == 4) {
 				return new ResponseEffectsOnly("8 hours", "Loiter in this area for the next eight hours.") {
 					@Override
 					public void effects() {
-//						Main.mainController.openPhone();
-						Main.game.getTextStartStringBuilder().append(
-								"<p style='text-align:center;'>"
-										+ "<i>You spend the next eight hours loitering about, doing nothing in particular...</i>"
-								+ "</p>");
-						Main.game.endTurn(60*60*8);
-						Main.game.setContent(new Response("", "", Main.game.getDefaultDialogue()));
+						Loiter(60*8);
 					}
 				};
 				
 			} else if(index == 5) {
 				return new ResponseEffectsOnly("12 hours", "Loiter in this area for the next twelve hours.") {
 					@Override
 					public void effects() {
-//						Main.mainController.openPhone();
-						Main.game.getTextStartStringBuilder().append(
-								"<p style='text-align:center;'>"
-										+ "<i>You spend the next twelve hours loitering about, doing nothing in particular...</i>"
-								+ "</p>");
-						Main.game.endTurn(60*60*12);
-						Main.game.setContent(new Response("", "", Main.game.getDefaultDialogue()));
+						Loiter(60*12);
 					}
 				};
 			}
 			return null;
 		}
 
+		private void Loiter(int minutes) {
```

Comment:
Minor nitpick: function names should start with a lowercase letter.

Reply:
Done